app/components/Chart/BarChart.tsx

```diff
@@ -1,7 +1,8 @@
 import { BarChart, Bar, CartesianGrid, Cell, XAxis, YAxis } from 'recharts';
+import type {
+  DistributionDataPoint} from 'app/components/Chart/utils';
 import {
-  CHART_COLORS,
-  DistributionDataPoint,
+  CHART_COLORS
 } from 'app/components/Chart/utils';
```

Comment:
Does it really format like that 👀

Reply:
Oh god, that's horrible. I might have not run prettier 😬